frontend/src/components/PackingList/PackingList.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styles from "./PackingList.module.css";
+import { Button } from "@material-ui/core";
+import { withStyles } from "@material-ui/core";
+
+import Item from "./Item/Item";
+
+const AddItemButton = withStyles(() => ({
+    root: {
```

Comment:
Is there a way of having this in a css style sheet to make things cleaner

Reply:
good idea, I made `PackingList.styles.js` and `Item.styles.js` files to pull this MUI styling out.